pihole-google.txt

```diff
@@ -7688,3 +7688,24 @@ www.tokyo.jibemobile.com
 www.viper.jibemobile.com
 www.watermelon.jibemobile.com
 yellowstone.jibemobile.com
+# More
+300zlf-dnsotls-ds.metric.gstatic.com
+cdn.fastclick.net
+e.crashlytics
+gstaticadssl.l.google.com
+KQu0cL-dnsotls-ds.metric.gstatic.com
+media.fastclick.net
+optimizationguide-pa.googleapis.com
+passwordsleakcheck-pa.googleapis.com
+peoplestackwebexperiments-pa.clients6.google.com
+pf-in-f188.1e100.net
+sa-in-f188.1e100.net
+tg-in-f188.1e100.net
+wa-in-f188.1e100.net
+wl-in-f188.1e100.net
+wq-in-f188.1e100.net
+www-google-analytics.l.google.com
```

Comment:
```suggestion
```

This one is already in the [regex](https://github.com/nickspaargaren/no-google#warning-important-regex-filters-not-included-in-blocklist).

Reply:
Done